Fix compilation error: conflicting types for 'locale_t'.

This is an excerpt from the debian patch ... 
http://http.debian.net/debian/pool/main/t/tvtime/tvtime_1.0.2-14.diff.gz

--- tvtime-1.0.2.orig/src/xmltv.c
+++ tvtime-1.0.2/src/xmltv.c
@@ -118,9 +118,9 @@
 typedef struct {
     const char *code;
     const char *name;
-} locale_t;
+} tvtime_locale_t;
 
-static locale_t locale_table[] = {
+static tvtime_locale_t locale_table[] = {
     {"AA", "Afar"},           {"AB", "Abkhazian"},      {"AF", "Afrikaans"},
     {"AM", "Amharic"},        {"AR", "Arabic"},         {"AS", "Assamese"},
     {"AY", "Aymara"},         {"AZ", "Azerbaijani"},    {"BA", "Bashkir"},
@@ -168,7 +168,7 @@
     {"XH", "Xhosa"},          {"YO", "Yoruba"},         {"ZH", "Chinese"},
     {"ZU", "Zulu"} };
 
-const int num_locales = sizeof( locale_table ) / sizeof( locale_t );
+const int num_locales = sizeof( locale_table ) / sizeof( tvtime_locale_t );
 
 /**
  * Timezone parsing code based loosely on the algorithm in
